frontend/src/pages/playback/video/index.vue

```diff
@@ -1,10 +1,111 @@
 <template>
-  <v-main class="fill-height" :class="{ 'cursor-none': !osd }">
-    <div class="fullscreen-video-container fill-height" />
-    <osd-player
-      v-model="osd"
-      :is-fullscreen="fullscreen.isFullscreen.value"
-      @toggle-fullscreen="toggleFullscreen" />
+  <v-main
+    class="fullscreen-video-container fill-height"
+    :class="{ 'cursor-none': !overlay }"
+    @mousemove="handleMouseMove"
+    @touchend="handleMouseMove">
+    <v-overlay
+      v-model="overlay"
+      contained
+      scrim="transparent"
+      width="100%"
+      height="100%">
+      <div
```

Comment:
Is this a strict copy of OsdPlayer.vue into this file or are there more substantial changes?

Reply:
@aweebs A few changes:

* Playback buttons use the new refactored components
* Event handling between the page and the component has been removed (no longer necessary in the same page)

But for the rest, it's strictly the same thing.